commands/link/__tests__/__fixtures__/with-contents/packages/package-1/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@test/package-1",
+  "version": "1.0.0",
+  "publishConfig": {
+    "directory": "dist"
```

Comment:
Also, I think you'll also want to run a test on the _existing_ fixtures with `--contents build` (or something), to distinguish it from the `publishConfig.directory` behavior.

Reply:
Thanks, this exposed some missing functionality with `link` and `bootstrap` so they respected `--contents`